index.js

```diff
@@ -1,42 +1,60 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1;
+counter = 2; //re-assign values
+console.log(counter) 
 // // => 2
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
+console.log("%cQuestion 2", "color: red") 
+const name = "Chris" //when using const, cant change/overwrite name. use const when you know for a fact 
+//the variable wont be changed
+console.log(name)//name is depracated. In the past in js library name was prob used. can be ignored
+// => "Chris"
 // console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
+// name = "Not Chris" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+function drinkWater(currentThirstLevel){ //use camelCase
```

Comment:
Great! One way to condense this and not lose any readability would be to consolidate lines 33 and 35 to:
```js
return currentThirstLevel - 1
```

- If we don't have the `return` and we try to use this function for getting a value, it will not work as expected as demonstrated in the following example:
```javascript
let myThirstLevel = drinkWater(7)
console.log(myThirstLevel) // undefined
```
- if we do have the `return` though, the same exact code will yield different results as shown below:
```javascript
let myThirstLevel = drinkWater(7)
console.log(myThirstLevel) // 6
```